addons/resource/models/resource.py

```diff
@@ -204,6 +204,16 @@ def copy(self, default=None):
             default.update(name=_('%s (copy)') % (self.name))
         return super(ResourceCalendar, self).copy(default)
 
+    @api.constrains('attendance_ids')
+    def _check_attendance_ids(self):
+        for resource in self:
+            if resource.two_weeks_calendar:
+                even_week_seq = resource.attendance_ids.filtered(lambda att: att.display_type == 'line_section' and att.week_type == '0')
```

Comment:
I would have done it like:
```python
if False in resource.attendance_ids.mapped('week_type'):
    raise.....
```

Reply:
week_type is not false.
If two_weeks_calendar is true, week_type is always set, see:
https://github.com/odoo/odoo/blob/2922d359f550f0d9a52845182fb4ab331b5349d7/addons/resource/models/resource.py#L308

Reply:
but you showed me it empty?

Reply:
ah yeah, but now you add it in the view...

Reply:
yes, it's my solution for the firs issue, that creates this inconsistency :(

Reply:
I don't see much better than the current version, this should probably work:

```py
if resource.attendance_ids and resource.attendance_ids.sorted('sequence')[0].display_type != 'line_section':
```